packages/tsx/tsx.loader.mjs

```diff
@@ -61,13 +61,12 @@ export { resolveTSX as resolve };
  * @argument {FileURL} url
  */
 async function loadTSX(url, ctx, nextLoad) {
-	// @ts-ignore https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71492
+	// @ts-ignore https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71493
```

Comment:
This fix could land today (I forgot about that PR); if so, these can be removed.

Reply:
so wait